src/main/java/org/dita/dost/ant/ExtensibleAntInvoker.java

```diff
@@ -316,10 +316,16 @@ public static Job getJob(final Project project) {
                 project.addReference(ANT_REFERENCE_JOB, job);
             }
         } else {
+            XMLUtils xmlUtils = project.getReference(ANT_REFERENCE_XML_UTILS);
```

Comment:
Store `XMLUtils` into Ant project reference, because we want to reuse the same Saxon `Processor` everywhere. This allows reusing the same name pool etc. that's needed for in-memory `Store` implementation.

Reply:
Just curious -- in the past we had memory crashes in steps where Saxon ended up reading 5,000 or 10,000 topics (like the `mappull` step). Is reusing the same Saxon `Processor` going to have any impact on that sort of case, either better or worse, or is there no change?

Reply:
No, the `Processor` class is what maintains the name pool etc. The `XsltExecutable` is the one that would be a problem and we don't reuse it by default.